addons/sourcemod/scripting/vsh/stocks.sp

```diff
@@ -601,6 +601,13 @@ stock int TF2_CreateAndEquipWeapon(int iClient, int iIndex, char[] sClassnameTem
 		DispatchSpawn(iWeapon);
 		SetEntProp(iWeapon, Prop_Send, "m_bValidatedAttachedEntity", true);
 		
+		if (HasEntProp(iWeapon, Prop_Send, "m_hExtraWearable"))
+		{
+			int iExtraWearable = GetEntPropEnt(iWeapon, Prop_Send, "m_hExtraWearable");
+			if (iExtraWearable != INVALID_ENT_REFERENCE)
+				SetEntProp(iExtraWearable, Prop_Send, "m_bValidatedAttachedEntity", true);
+		}
```

Comment:
`m_hExtraWearable` should exist in all weapons and not wearables, so it can be moved to below code after wearable check, without needing `HasEntProp` check